mdtraj/geometry/rdf.py

```diff
@@ -67,7 +70,10 @@ def compute_rdf(traj, pairs=None, r_range=None, bin_width=0.005,
         r_range = np.array([0.0, 1.0])
     r_range = ensure_type(r_range, dtype=np.float64, ndim=1, name='r_range',
                           shape=(2,), warn_on_cast=False)
-    bins = np.arange(r_range[0], r_range[1], bin_width)
+    if n_bins is not None:
```

Comment:
Actually, I'd go back to `if n_bins`.  `if n_bins is not None` is preferred to `if n_bins != None`, but you actually changed the meaning of your code with this change (if that's intentional, apologies).

I would intuitively think that any value for `n_bins` that evaluated to `False` (e.g., `0`, `False`, etc.) would fall back to `bin_width`.  And the error message will probably be a little strange if someone feeds an illegal number of bins (like <0 or 0)?